sdk/cosmos/dev.md

```diff
@@ -0,0 +1,56 @@
+# Dev notes
+
+## Checking out the code
+
+The SDK is open source and is available here [sdk](sdk/).
+
+Clone the Repo
+
+```bash
+git clone https://github.com/Azure/azure-cosmosdb-java.git
+cd azure-cosmosdb-java
+```
+
+## How to Build from Command Line
+
+- Run the following maven command to build:
+
+```bash
+maven clean package -DskipTests
+```
+
+## How to generate directory structure for publishing
+
+- Run the following maven command to collect the jars needed for publishing
+
+```bash
+mvn antrun:run -N
```

Comment:
this was used prior to merging to central repo.
Now the central repo should provide a way of releasing the sdk.

so the ant job is not relevant anymore. this section "How to generate directory structure for publishing" can be removed.

Reply:
removed.